app/src/main/java/com/example/lapnguyen/weather_3/api/GPSTracker.java

```diff
@@ -0,0 +1,161 @@
+package com.example.lapnguyen.weather_3.api;
+
+/**
+ * Created by lapnguyen on 16/06/2017.
+ */
+
+import android.Manifest;
+import android.app.AlertDialog;
+import android.app.Service;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.location.Location;
+import android.location.LocationListener;
+import android.location.LocationManager;
+import android.os.Bundle;
+import android.os.IBinder;
+import android.provider.Settings;
```

Comment:
ERROR: Unused import: android.provider.Settings. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports